jormungandr-lib/src/interfaces/certificate.rs

```diff
@@ -141,6 +141,12 @@ impl Readable for Certificate {
                 let cert = certificate::VoteTally::read(buf)?;
                 Ok(Certificate(certificate::Certificate::VoteTally(cert)))
             }
+            9 => {
+                let cert = certificate::EncryptedVoteTally::read(buf)?;
+                Ok(Certificate(certificate::Certificate::EncryptedVoteTally(
+                    cert,
+                )))
+            }
```

Comment:
And this is why those ids are bad. Rust does not complain if it doesn't find them.